### PR TITLE
chore(readme): improve what is working section

### PR DESCRIPTION
**Pull request by reslear**
can be later, based on this list it will be possible to check nuxt 3 support ☺️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,15 +52,24 @@ Features:
...
+- [ ] [auth-next](https://auth.nuxtjs.org/)
 - [ ] Composition API
 - [ ] Tailwindcss
 
+<!--
+Community Modules:
+- [ ] [svg-sprite-module] (https://github.com/nuxt-community/svg-sprite-module)
+-->
+
+Vite Plugins:
```
Comment (pi0) [inline README.md:70]: Ideally there should be nuxt modules that support both webpack and vite. Adding vite-only support makes fragmentation in ecosystem and also we still use webpack for production build... So i would suggest not including this list

File: README.md [hunk 2]
```diff
@@ -52,15 +52,24 @@ Features:
 - [x] Page [middleware](https://nuxtjs.org/docs/2.x/directory-structure/middleware/)
 - [x] Basic [jsx](https://vuejs.org/v2/guide/render-function.html#JSX) (pass `h` as first argument)
 
-Modules:
+Official Modules:
```
Comment (pi0) [inline README.md:55]: They are both official and community. It is mostly list of popular modules

File: README.md [hunk 3]
```diff
@@ -52,15 +52,24 @@ Features:
...
 
 - [X] [http](https://http.nuxtjs.org/)
 - [x] [axios](https://axios.nuxtjs.org/)
 - [X] [i18n](https://i18n.nuxtjs.org/)
 - [x] [content](https://content.nuxtjs.org/)
+- [ ] [auth-next](https://auth.nuxtjs.org/)
```
Comment (pi0) [inline README.md:61]: Module name is `auth`. Before adding to list, can you please help preparing a small repo for using vite and auth to investigate problems? (we can use a github issue)

File: README.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ That's it! Now you can enjoy super fast `nuxt dev` experience with Vite!
 
 **[📖 Read documentation for more](https://vite.nuxtjs.org)**
 
-## ✔️ What is working?
+## ✔️ What is working and tested?
```
Comment (pi0) [inline README.md:42]: Adding tested at current stage is not correct since they features are basically working but not covering all edge-cases

**Comment by reslear** [comment]
@pi0 you're right)
